docker-compose.yml

```
@@ -261,3 +261,11 @@ services:
     volumes:
       - "./fossildb-dev/data:/fossildb/data"
       - "./fossildb-dev/backup:/fossildb/backup"
+
+  # Redis
+  redis:
+    image: redis:5.0.3
```

Let's use the same as in k8:
```suggestion
    image: redis:5.0-alpine
```